docs/source/get-started/build-and-run-examples.rst

```diff
@@ -76,6 +76,8 @@ basic usage scenarios of |short_name| with DPCPP. Go to
 #. Set up the compiler environment for |dpcpp|.
    See |dpcpp_gsg|_ for details.
 
+#. Draft:For static applications and examples is necesasry to source oneMKL.
```

Comment:
The instruction on how to set up MKL environment is needed here.
Please add something like this. Maybe separate tabs for Linux and Windows are needed.
```suggestion
#. Set up oneMKL environment in case of static linking:
    .. code-block:: bash
    source mkl/latest/env/vars.sh intel64 ilp64
```